src/Scripting/Core/ScriptOptions.cs

```diff
@@ -63,6 +63,11 @@ private static ImmutableArray<MetadataReference> GetDefaultMetadataReferences()
                 "System.Threading.Tasks",
                 "System.Threading.Tasks.Parallel",
                 "System.Threading.Thread",
+                "System.Linq",
+                "System.Linq.Expressions",
+                "System.Runtime.Numerics",
+                "System.Dynamic.Runtime",
+                "Microsoft.CSharp",
```

Comment:
This roughly matches the surface area of DLLs in csi.rsp. Notably, csi.rsp has a MS.CSharp reference while this did not, so you couldn't use dynamic.

---
In reply to: [110797808](https://github.com/dotnet/roslyn/pull/18471#discussion_r110797808) [](ancestors = 110797808)